project/plugins.sbt

```diff
@@ -1 +1,5 @@
 addSbtPlugin("de.heikoseeberger" % "sbt-header" % "1.6.0")
+
+addSbtPlugin("com.github.gseitz" % "sbt-protobuf" % "0.4.0")
+
+libraryDependencies += "com.github.os72" % "protoc-jar" % "3.3.0.1"
```

Comment:
This version is using protoc 3.3.0 but applying a fix to the protoc-jar project
https://github.com/os72/protoc-jar/blob/v3.3.0.1/RELEASE_NOTES.md#3301-22-jun-2017